### PR TITLE
Create Initial Layout

### PR DESCRIPTION
**Pull request by cmriker**
**Overview:**
* `<App>` has been updated to use the new views
* `<RouteView>` gets route name, parameters, and available methods to display
* `<RequestTestSelector>` creates two `<select>`s to allow someone to pick a method type and route name, and a button for a future `<form>` submission
    * This component will get wrapped in a `<form>` next which will have an `action` equal to the selected route path and a `method` equal to the selected method type
* There was nothing specified in the README on preferred testing strategies, so I installed the modules I'm used to working with to start out.
    * I roughly followed TDD for changes to `App.js` and when I created `RouteView.js` but after a lot of wasted time trying to figure out which piece I missed when configuring things and writing the `test` script, I abandoned them for now to save time. 

**Screenshots:**
The new layout using the example `todos.json`
image

**Side Notes:**
Here is a screenshot of the syntax error I am seeing when trying to run `yarn test`
image

### CODE REVIEW

File: app/containers/App.js [hunk 1]
```diff
@@ -2,23 +2,25 @@ import React from 'react';
 import { connect } from 'react-redux';
 
 import style from './App.css';
+import RouteView from './RouteView';
+import RequestTestSelector from './RequestTestSelector';
 
 class App extends React.Component {
   render() {
     const { swagger } = this.props;
+    const paths = Object.entries(swagger.paths);
```
Comment (cmriker) [inline app/containers/App.js]: On my list of refactors/todos if there is time is to write a util that does all the transformations to the swagger initially, to make things inside each component much less messy (no more `Object.entries` all over each file)

File: package.json [hunk 2]
```diff
@@ -3,7 +3,8 @@
   "version": "0.0.1",
   "description": "Coding challenge",
   "scripts": {
-    "start": "node scripts/start"
+    "start": "node scripts/start",
+    "test": "cross-env NODE_ENV=development mocha --require babel-core/register --colors --recursive test"
```
Comment (cmriker) [inline package.json]: I tried lost of variations for this command, including trying to use `--compilers js:babel-core/register` instead of `--require babel-core/register` and trying to run `babel` on the codebase before running `mocha`. I got frustrated with different configurations and sort of abandoned this command midway through. I am still going to be adding tests as I find them useful, but following the TDD cycle is...impossible if I can't get the tests to run at all.

**tl;dr:** tests are not working and I still don't know why

**Comment by cmriker** [comment]
All the changes look how I expected. I probably could have left out the broken `test` script in `package.json` but I'll leave it and try to troubleshoot it more later if I have time.
